Web/JQuerry-1/Assignment1/JavaScript.js

```diff
@@ -0,0 +1,15 @@
+function show() {
+				var showValue=document.getElementById('input');
+				showValue.innerHTML="My name is Anshita Arya";
+			}
+			function hide() {
+				var showValue=document.getElementById('input');
+				showValue.innerHTML="";
+			}
+			function showOrHide() {
+				var showValue=document.getElementById('input');
```

Comment:
ACTION: We need to modify button text. As per question "Show or hide depending on the span state, that is if span is visible it should show “Hide” and if span is hidden, it should show “Show"".